README.rst

```diff
@@ -31,6 +31,13 @@ To install using pip
    
    pip install urwid
 
+For advanced functionality extra requirements need to be installed.
+Example for ZeroMQ event loop and LCD display:
```

Comment:
is there a standard way to get a list of optional dependencies and their descriptions or should we add that here/in the docs?

Reply:
They're described in pyproject.toml... No, don't duplicate in docs, will get outdated

Reply:
Pip feature request is multiple years old, even solution with API is already added to the issue.
At this moment optional dependencies are not visible without API